src/languages/es.js

```diff
@@ -492,16 +492,16 @@ export default {
         growlMessageOnSave: 'Su tarteja de débito se agregó correctamente',
         expensifyPassword: 'Contraseña de Expensify',
         error: {
-            invalidName: 'Por favor ingresa un nombre válido',
-            addressZipCode: 'Por favor ingresa un código postal válido',
-            debitCardNumber: 'Ingresa un número de tarjeta de débito válido',
-            expirationDate: 'Por favor introduce una fecha de vencimiento válida',
-            securityCode: 'Ingresa un código de seguridad válido',
-            addressStreet: 'Ingresa una dirección de facturación válida que no sea un apartado postal',
-            addressState: 'Por favor seleccione un estado',
-            addressCity: 'Por favor ingresa una ciudad',
-            genericFailureMessage: 'Se produjo un error al agregar su tarjeta. Vuelva a intentarlo',
-            password: 'Por favor ingresa tu contraseña de Expensify',
+            invalidName: 'Introduce un nombre válido',
+            addressZipCode: 'Introduce un código postal válido',
+            debitCardNumber: 'Introduce un número de tarjeta de débito válido',
+            expirationDate: 'Introduce una fecha de expiración válida',
+            securityCode: 'Introduce un código de seguridad válido',
+            addressStreet: 'Introduce una dirección de facturación válida que no sea un apartado postal',
+            addressState: 'Seleccione un estado',
+            addressCity: 'Introduce una ciudad',
+            genericFailureMessage: 'Se produjo un error al agregar tu tarjeta. Vuelve a intentarlo',
+            password: 'Introduce tu contraseña de Expensify',
```

Comment:
Why less cordial in spanish, not saying please?

Reply:
It sounds so weird to say please that many times in Spanish just for error messages.

Reply:
Like it doesn't sound rude or anything at all if we skip the please

Reply:
I don't know, same thing could be said about the english phrases and we still have the `please` there 🤷

Reply:
Do you want to add Por favor to every error or not? I prefer without it 🤷‍♀️

Reply:
I think we either add it to spanish or remove it from english, no? The tone of the copy should be the same in both languages

Reply:
I will add it to all of these then 😄